thirdparty/build-thirdparty.sh

```diff
@@ -431,7 +431,7 @@ build_lzo2() {
 build_bzip() {
     check_if_source_exist $BZIP_SOURCE
     cd $TP_SOURCE_DIR/$BZIP_SOURCE
-
+    CFLAGS="${CFLAGS} -fPIC"
```

Comment:
this won't work.

Reply:
no need CFLAGS because we have patch the bzip source code.